src/compute/src/logging/compute.rs

```diff
@@ -113,6 +113,11 @@ pub enum ComputeEvent {
         /// Operator index
         operator: usize,
     },
+    /// All operators of a dataflow have shut down.
+    DataflowShutdown {
+        /// Timely worker index of the dataflow.
+        index: usize,
```

Comment:
Maybe we could rename this to `dataflow_id`, because `index` seems to be quite generic.

Reply:
It does match the pattern we have with `Export::id` and `Frontier::id` where the field name assumes the context of the event type. But I don't have strong feelings about this. Happy to change it to `dataflow_index` to match `Export::dataflow_index`.

(We have some confusion going on between the ID of the operator that is the dataflow's top-level scope and the first component of the dataflow's address. Both are called "ID" at times, but only the second thing is ever called "index" I think.)

Reply:
Renamed to `dataflow_index`.